spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AboutController.java

```diff
@@ -93,8 +94,17 @@
 
 	private LauncherRepository launcherRepository;
 
+	@Value("${spring.cloud.dataflow.grafana-info.url:#{null}}")
+	private String grafanaUrl;
+
+	@Value("${spring.cloud.dataflow.grafana-info.token:#{null}}")
+	private String grafanaToken;
+
+	@Value("${spring.cloud.dataflow.grafana-info.refreshInterval:15}")
```

Comment:
These should really go via configuration props as using a @Value like will have issues. i.e. `--spring.cloud.dataflow.grafana-info.refresh-interval=20` doesn't work.

Reply:
Those properties and the related `GrafanaInfo` are not used internally by the SCDF server but only as a mean to configure the SCDF-UI. So perhaps using spring configuration props might be an overkill? 

For i've successfully tested with properties like:
```
--spring.cloud.dataflow.grafana-info.url=http://localhost:3000
--spring.cloud.dataflow.grafana-info.refreshInterval=30
```

Not sure if the `xxx.refresh-interval` relaxation would work. Probably not.

Reply:
@jvalkeal, as suggested I've replaced the ad-hoc `@Value` properties with proper GrafanaInfoProperties object.  This provides a cleaner separation of concerns.